test-plugin/build.gradle.kts

```diff
@@ -40,18 +40,28 @@ check(gradleTest.files.single().isGradleApi())
 
 dependencies {
   // Do not use gradleApi() as it forces Kotlin 1.4 on the classpath
-  compileOnly("dev.gradleplugins:gradle-api:6.9")
+  compileOnly("dev.gradleplugins:gradle-api:7.6") {
+    exclude("org.apache.ant")
```

Comment:
Got the same thing here with duplicate `org.apache.ant` class but I'm still unclear where the other one is. Or maybe it's because `org.apache.ant` is also present in `addProgramFiles`?